dev/src/util.ts

```diff
@@ -92,3 +96,24 @@ export function isEmpty(value: {}): boolean {
 export function isFunction(value: unknown): boolean {
   return typeof value === 'function';
 }
+
+/**
+ * Determines whether the provided error is considered permanent for the given
+ * RPC.
+ *
+ * @private
+ */
+export function isPermanentRpcError(
+  err: GoogleError,
+  methodName: string,
+  config: ClientConfig
+): boolean {
+  const serviceConfig = config.interfaces!['google.firestore.v1.Firestore']!;
```

Comment:
Just a heads up, this config file will go away soon. By "soon" here I mean half-a-year-ish time. When backend starts supporting returning this configuration via DNS (TXT record I guess), we'll start using it and will stop generating those JSON configs.

Reply:
Got it. I will find some clever way to get this configuration in 6 months from now.

Or rather - I will find a clever way to push this code into GAX.